src/impls/mod.rs

```diff
@@ -0,0 +1,10 @@
+#[cfg(feature = "alloc")]
+mod allocs;
```

Comment:
Since this refers to the `alloc` crate, it should probably be called `alloc`
```suggestion
mod alloc;
```

Reply:
this would name-clash with the `extern crate alloc`, if the `alloc` feature is activated: https://github.com/knurling-rs/defmt/blob/fa902a95c5748fbd55f3ad173fe3d9d5873c68a8/src/lib.rs#L18-L19

Reply:
Hmm, but this is in `impls`, not in the root, so shouldn't it be fine?

Reply:
running `cargo xtask test-host` will error with multiple `error[E0433]: failed to resolve: could not find 'borrow' in 'alloc'`

Reply:
Huh, okay, then let's go with `alloc_` maybe?

Reply:
I tried to match the plural I used for `primitives`, `tuples`, `arrays`, but I am also good with this naming 😄